validator/src/lib.rs

```diff
@@ -33,55 +36,55 @@ fn redirect_stderr(filename: &str) {
     }
 }
 
-// Redirect stderr to a file with support for logrotate by sending a SIGUSR1 to the process.
-//
-// Upon success, future `log` macros and `eprintln!()` can be found in the specified log file.
-pub fn redirect_stderr_to_file(logfile: Option<String>) -> Option<JoinHandle<()>> {
-    // Default to RUST_BACKTRACE=1 for more informative validator logs
-    if env::var_os("RUST_BACKTRACE").is_none() {
-        env::set_var("RUST_BACKTRACE", "1")
-    }
```

Comment:
If we're going for minimal changes, we still want this right ? Even if not, we still want/need this piece right ?